bindings/pyroot_experimental/PyROOT/test/root_module.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+
+
+class ROOTModule(unittest.TestCase):
+    def test_import(self):
+        """
+        Test import
+        """
+        import ROOT
+        self.assertEqual(ROOT.__name__, "ROOT")
+
+
+    def test_relative_import(self):
+        """
+        Test relative import
+        """
+        from ROOT import TH1F
+        self.assertEqual(TH1F.__name__, "TH1F")
+        from ROOT import TH1F as Foo
+        self.assertEqual(Foo.__name__, "TH1F")
+        self.assertEqual(TH1F, Foo)
+
+
+    def test_version(self):
+        """
+        Test __version__ property
+        """
+        import ROOT
+        v = ROOT.__version__
+        self.assertTrue(type(v) == str)
+        self.assertIn("/", v)
+        self.assertIn(".", v)
```

Comment:
This is great, thanks! Should we also compare here the version we obtain from `ROOT.__version__` to the one `gROOT.GetVersion` gives?

Reply:
Done!